py/etrade-plan-pdf-tx-extract

```diff
@@ -527,14 +528,20 @@ def find_and_apply_sell_to_cover_trade_set(benefit: BenefitEntry, trade_confs: l
          if n_shares == benefit.sell_to_cover_shares:
             if matching_trades is not None:
                if set(matching_trades) != set(trades):
-                  print(f"Error: Multiple trade combinations near {benefit.acquire_tx_date} "
-                         "could potentially constitute the sale", file=sys.stderr)
-                  return []
+                  errors.append(f"Error: Multiple trade combinations near {benefit.acquire_tx_date} "
+                                f"could potentially constitute the sale {benefit.filename}")
+                  matching_trades = None
```

Comment:
I think this needs a bit of comment insight. As-is, I think this is now non-deterministic. You could in theory see your trade combinations in an arbitrary order, so taking essentially the last-match-standing seems weird.

Is there a specific, probable case that this solves? Can we get there more deterministically?

Reply:
In fact, I think it must have been left around from debugging last year. I don't have anything that needs it currently, at least. I set up my test cases, reverted the change, and it all still passed.

Reply:
See https://github.com/tsiemens/acb/pull/27/files/8c9130545c7b3844fe4f5d224e455aa8464a93e3..0a57482f70583e37e5699598bbec3e6fc917790b